documentation/configuration.md

```diff
@@ -1401,3 +1401,95 @@ The following example shows the `Limits` portion of a collection rule that has t
+
+| Name | Section | Type | Applies To |
+|---|---|---|---|
+| `Egress` | `Actions` | string | [CollectDump](#collectdump-action), [CollectGCDump](#collectgcdump-action), [CollectTrace](#collecttrace-action), [CollectLogs](#collectlogs-action) |
+| `SlidingWindowDuration` | `Triggers` | TimeSpan? | [AspNetRequestCount](#aspnetrequestcount-trigger), [AspNetRequestDuration](#aspnetrequestduration-trigger), [AspNetResponseStatus](#aspnetresponsestatus-trigger), [EventCounter](#eventcounter-trigger) |
+| `RequestCount` | `Triggers` | int | [AspNetRequestCount](#aspnetrequestcount-trigger), [AspNetRequestDuration](#aspnetrequestduration-trigger) |
+| `ResponseCount` | `Triggers` | int | [AspNetResponseStatus](#aspnetresponsestatus-trigger) |
+| `ActionCount` | `Limits` | int | [Limits](#limits) |
+| `ActionCountSlidingWindowDuration` | `Limits` | TimeSpan? | [Limits](#limits) |
+| `RuleDuration` | `Limits` | TimeSpan? | [Limits](#limits) |
+
+### Example
+
+The following example includes a default egress provider that corresponds to the `FileSystem` egress provider named `artifacts`. The first action, `CollectDump`, is able to omit the `Settings` section by using the default egress provider. The second action, `CollectGCDump`, is using an egress provider other than the default, and specifies that it will egress to an `AzureBlobStorage` provider named `monitorBlob`.
+
+<details>
+  <summary>JSON</summary>
+
+  ```json
+  {
+    "Egress": {
+      "AzureBlobStorage": {
+        "monitorBlob": {
+          "accountUri": "https://exampleaccount.blob.core.windows.net",
+          "containerName": "dotnet-monitor",
+          "blobPrefix": "artifacts",
+          "accountKeyName": "MonitorBlobAccountKey"
+        }
+      },
+      "Properties": {
+        "MonitorBlobAccountKey": "accountKey"
+      },
+      "FileSystem": {
+        "artifacts": {
+          "directoryPath": "/artifacts",
+          "intermediateDirectoryPath": "/intermediateArtifacts"
+        }
+      }
+    },
+    "CollectionRuleDefaults": {
+      "Actions": {
+        "Egress": "artifacts"    
+      }
+    },
+    "CollectionRules": {
+      "HighRequestCount": {
+        "Trigger": {
+          "Type": "AspNetRequestCount",
+          "Settings": {
+            "RequestCount": 10
+          }
+        },
+        "Actions": [
+          {
+            "Type": "CollectDump"
+          },
+          {
+            "Type": "CollectGCDump",
+            "Settings": {
+              "Egress": "monitorBlob"
+            }
+          }
+        ]
+      }
+    }
+  }
+  ```
+</details>
+
+<details>
+  <summary>Environment Variables</summary>
+  
+  ```bash
+  export DotnetMonitor_Egress__AzureBlobStorage__monitorBlob__accountUri="https://exampleaccount.blob.core.windows.net"
```

Comment:
Would this be more useful as yaml?

Reply:
I guess I'm not sure if you've seen it, but for QFW I added a lot of sections like this throughout configuration: https://github.com/dotnet/dotnet-monitor/pull/1686